lib/rules/jquery-ember-run.js

```diff
@@ -43,28 +33,83 @@ module.exports = {
       context.report(node, ERROR_MESSAGE);
     };
 
-    return {
-      CallExpression(node) {
-        const callee = node.callee;
-        const fnNodes = utils.findNodes(node.arguments, 'ArrowFunctionExpression');
+    let importedEmberName;
+    let importedBindName;
 
-        if (isJqueryUsed(callee) && fnNodes.length > 0) {
-          fnNodes.forEach((fnNode) => {
-            const fnBody = fnNode.body.body;
-            const fnExpressions = utils.findNodes(fnBody, 'ExpressionStatement');
+    function checkJqueryCall(node) {
```

Comment:
Thanks! I got stuck on this bit.. Wasn't too sure about how to go about testing this in an easy way..

I started looking at using `lib/utils/traverser.js` but got a little lost 😓

Reply:
Yeah I'm pretty unhappy with how ugly this `checkJqueryCall` is currently though...